lively.ide/code-search.cp.js

```diff
@@ -159,9 +159,7 @@ Regular expression should be given without quotes or literal mode slashes.',
             listMaster: SystemList,
             items: [
               'in loaded modules',
-              'in loaded and unloaded modules',
-              'in parts',
-              'in worlds'
```

Comment:
In case you are fine with removing and then readding, instead of leaving it as a "live reminder" - I wrote it down and would proceed with removing it :see_no_evil: